standard/requirements/core/REQ_record_update_date.adoc

```diff
@@ -2,6 +2,6 @@
 [width="90%",cols="2,6a"]
 |===
 ^|*Requirement {counter:req-id}* |*/req/core/record_update_date*
-^|A |A WCMP record SHALL provide a `+properties.recordUpdated+` property.
+^|A |A WCMP record SHOULD provide a `+properties.recordUpdated+` property.
```

Comment:
If this is a SHOULD then this requirement turns into a recommendation.  iirc we wanted to have a `recordUpdated` property so as to be able to detect updates over time?

Reply:
I changed back to SHALL. We can discuss again later if we want